README.md

```diff
@@ -1,5 +1,5 @@
 
-[![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
+Staging [![Build Status](https://travis-ci.org/flexion/hud-disaster-data.svg?branch=master)](https://travis-ci.org/flexion/hud-disaster-data)
```

Comment:
A fine solution. Maybe better ones exist, but that's enough for me to feel good about this. Thanks!